packages/ebayui-core-react/src/ebay-menu/menu.tsx

```diff
@@ -148,6 +148,7 @@ const EbayMenu: FC<EbayMenuProps> = ({
                         onClick: onItemClick = () => {},
                         onFocus: onItemFocus = () => {},
                         onKeyDown: onItemKeyDown = () => {},
+                        disabled,
```

Comment:
The `disabled` needs to also be passed down to the child as it was before from `itemRest`. Can you add a test for this case also? it should have an `aria-disabled` the item.